### DISCUSSION TITLE
Blobfuse2 - azurite with https, skip certificate check

### DISCUSSION THREAD

**Discussion by marwan-husseisi** [Azure/azure-storage-fuse#1440] [Q&A] [answered]
Hi, I'm trying to mount an emulated azurite running https(this is required, http not relevant - it works), and I need the mount operation to skip certificate check, similar to no_check_certificate for s3fs, can't figure out how to do that, is this supported? if so, how can I get it to work?
Thanks

**Comment by bhansalivikas** [comment, marked as answer]
There is no direct flag to skip the certificate check in blobfuse. Either you can add the self-signed certificate to standard Linux path or make some changes at system level if possible, to skip those checks. Due to security first policy, we do not provide any option to skip the certificate check.

**Reply by marwan-husseisi** [reply]
Thank you for the quick reply
